src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -100,52 +102,53 @@ The .NET Foundation licenses this file to you under the MIT license.
       <StaticSslLibs Include="-Wl,-Bdynamic" Condition="'$(StaticExecutable)' != 'true'" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(TargetOS)' == 'osx'">
+    <ItemGroup Condition="'$(_IsApplePlatform)' == 'true'">
       <NativeFramework Include="CoreFoundation" />
       <NativeFramework Include="CryptoKit" />
       <NativeFramework Include="Foundation" />
       <NativeFramework Include="Security" />
-      <NativeFramework Include="GSS" />
+      <!-- The library builds don't reference the GSS API on tvOS builds. -->
+      <NativeFramework Condition="'$(TargetOS)' != 'tvos' and '$(TargetOS)' != 'tvossimulator'" Include="GSS" />
```

Comment:
nit: Use `$(TargetOS.StartsWith('tvos'))`

Reply:
I find the explicit version a bit easier to read. `!$(TargetOS.StartsWith('tvos'))` (`!` first) should work but I didn't try it.

Reply:
I also find the explicit version easier to reason about.